recipe/patches/0001-Blacklist-well-known-protos.patch

```diff
@@ -0,0 +1,26 @@
+From 9e6d16913eedc72aad7ced7f9cdf07374c84fc8f Mon Sep 17 00:00:00 2001
+From: "Uwe L. Korn" <uwe.korn@quantco.com>
+Date: Sun, 19 Nov 2023 20:50:29 +0000
+Subject: [PATCH] Blacklist well-known protos
```

Comment:
The problem here was that this would pull in the well-known protos twice. Thus, when importing tensorflow, we got errors about already registered definitions. I found the fix in https://github.com/google/riegeli/blob/c2bcb54934acd28eace78bd4a1bf008347592cc4/third_party/protobuf.patch#L64

I will merge this patch with above one that adds the toolchain during cleanup.